jwcrypto/jws.py

```diff
@@ -31,7 +42,7 @@
 }
 """Registry of valid header parameters"""
 
-default_allowed_algs = [
+jws_default_allowed_algs = [
```

Comment:
why the rename ?
the default_allowed_algs is an accessible property but not a pulic interface

Reply:
I agree, this is a bad decision, but then probably we should pass on yet the here (https://github.com/latchset/jwcrypto/blob/master/jwcrypto/jws.py#L482) parametr `self.allowed_algs` just like here (https://github.com/latchset/jwcrypto/blob/master/jwcrypto/jws.py#L288)